client/blocks/app-banner/utils.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Internal dependencies
+ */
+import {
+	startsWith,
+	get,
+	reduce,
+} from 'lodash';
+
+export const EDITOR = 'editor';
+export const NOTES = 'notifications';
+export const READER = 'reader';
+export const STATS = 'stats';
+export const ALLOWED_PAGE_TYPES = [ EDITOR, NOTES, READER, STATS ];
+export const ONE_WEEK_IN_MILLISECONDS = 604800000;
+export const ONE_YEAR_IN_MILLISECONDS = 31540000000;
+
+export function getAppBannerData( translate, pageType ) {
+	switch ( pageType ) {
+		case EDITOR:
+			return {
+				title: translate( 'Rich mobile publishing.' ),
+				copy: translate( 'A streamlined editor with faster, simpler image uploading? Check and mate.' ),
```

Comment:
Hmm, perhaps could we allow some child props in? We'd probably be able to use a component's translate vs from i18n directly.
```jsx
<AppBanner title={ title }>
{ copy }
</AppBanner>
```